### ISSUE TITLE
The BLEU score of translation model seems abnormal. The model doesn't seem to train effectively.

### ISSUE THREAD

**Issue by njuyexiangyu** [msr-fiddle/pipedream#63]
I trained translation model with the version 'GPUs=1' of pipedream. The loss during training is all right. After 10 epochs, the loss function drops normally until it converges. But when I run 'compute_bleu_scores.py' to calculate the BLUE score, I get the score of '0.01'. The model saved in each epochs gets very low score such as '0.01', '0.04'. When I output the translated sentences directly, the translation result is obviously incorrect. The translation results can be seen in the picture. There are many ',' and '.'. I want to know what's wrong, my reason or the source code. Thank you very much!!
![2021-01-11 11-14-21 的屏幕截图](https://user-images.githubusercontent.com/45187386/104144452-34595380-53fe-11eb-9038-34d61cb6d6aa.png)
